selfdrive/car/hyundai/interface.py

```diff
@@ -227,6 +227,18 @@ def get_params(candidate, fingerprint=gen_empty_fingerprint(), car_fw=[], disabl
       tire_stiffness_factor = 0.5
       ret.lateralTuning.pid.kiBP, ret.lateralTuning.pid.kpBP = [[0.], [0.]]
       ret.lateralTuning.pid.kpV, ret.lateralTuning.pid.kiV = [[0.25], [0.05]]
+    elif candidate == CAR.KIA_EV6:
+      ret.mass = 2055 + STD_CARGO_KG
+      ret.wheelbase = 2.9
+      ret.steerRatio = 16.
+      ret.safetyConfigs = [get_safety_config(car.CarParams.SafetyModel.noOutput),
```

Comment:
should probably use `SafetyModel.silent` for private radar bus just to make sure it doesn't interfere with anything?

Reply:
Boardd already does this if you don't specify a safetyconfig for a panda